src/index.tsx

```diff
@@ -713,51 +830,48 @@ export class App extends React.Component<any, AppState> {
             </section>
             <div />
           </div>
+          <div className="App-menu App-menu_bottom">
+            <Stack.Col gap={2}>
+              <section aria-labelledby="canvas-zoom-actions-title">
+                <h2 className="visually-hidden" id="canvas-zoom-actions-title">
+                  {t("headings.canvasActions")}
+                </h2>
+                <Island padding={1}>{this.renderZoomActions()}</Island>
+              </section>
+            </Stack.Col>
+          </div>
         </FixedSideContainer>
         <main>
           <canvas
             id="canvas"
             style={{
-              width: canvasWidth,
-              height: canvasHeight,
+              width: canvasDOMWidth,
+              height: canvasDOMHeight,
             }}
-            width={canvasWidth * window.devicePixelRatio}
-            height={canvasHeight * window.devicePixelRatio}
+            width={canvasWidth}
+            height={canvasHeight}
             ref={canvas => {
-              if (this.canvas === null) {
+              // canvas is null when unmounting
```

Comment:
From the [docs](https://reactjs.org/docs/refs-and-the-dom.html#adding-a-ref-to-a-dom-element):

> React will assign the current property with the DOM element when the component mounts, and assign it back to `null` when it unmounts.

This way, we can remove the `removeWheelEventListener` global variable and it's easier to read.

Reply:
that works